src/ExportConverters.mss

```diff
@@ -592,15 +592,11 @@ function ConvertSibeliusStructure (score) {
     {
         for each Bar b in s
         {
-            if (bar_to_staff.PropertyExists(b.BarNumber))
```

Comment:
Are these changes part of the plugin support?

Reply:
There are a few changes not strictly related to text, symbols and extensions that I made while working with the code.  That's the case here and in both the Run.mss files.  Some things are only for clarity and cosmetics (like here code that does the same thing, but a bit clearer; or inconsistent indentation in one Run.mss).  Some things are for easier development and use, like better file name defaults when exporting or making it possible to automatically calling the mocha tests after exporting the test set.